composer.json

```diff
@@ -8,10 +8,12 @@
     "aws/aws-sdk-php": "^3.147",
     "guzzlehttp/guzzle": "^7.0",
     "guzzlehttp/psr7": "^2.0",
-    "laminas/laminas-log": "^2.12",
+    "laminas/laminas-log": "^2.17",
     "league/flysystem": "^3.0.0",
     "league/flysystem-aws-s3-v3": "^3.0.0",
-    "lcobucci/jwt": "^5.0"
+    "lcobucci/jwt": "^5.0",
+    "laminas/laminas-servicemanager": "^3.22",
+    "laminas/laminas-stdlib": "^3.18"
```

Comment:
How did these pop in? It doesn't look like we directly depend on them?

Reply:
They were additional packages that the current version didn't support PHP 8.3 so had to explicitly upgrade them.

Reply:
I just redid this with `composer update laminas/laminas-log -W`, which resulted in the same changes but without adding these to the direct dependencies. I've added a commit of the diff, which resets the `composer.json` file and updates the PHP version in the lock file.

Reply:
Nice!